README.md

```diff
@@ -1,8 +1,9 @@
 # <img src="docs/images/logo_wide_v1.svg" alt="DCRDEX" width="250">
 
 [![Build Status](https://github.com/decred/dcrdex/workflows/Build%20and%20Test/badge.svg)](https://github.com/decred/dcrdex/actions)
-[![ISC License](https://img.shields.io/badge/license-Blue_Oak-007788.svg)](http://copyfree.org)
+[![ISC License](https://img.shields.io/badge/license-Blue_Oak-007788.svg)](https://blueoakcouncil.org/license/1.0.0)
 [![GoDoc](https://img.shields.io/badge/go.dev-reference-blue.svg?logo=go&logoColor=lightblue)](https://pkg.go.dev/decred.org/dcrdex)
+[![](https://img.shields.io/badge/license-ISC%20License-informational.svg)](https://github.com/decred/dcrd/blob/master/LICENSE)
```

Comment:
We don't want two license badges.  We can have a link in the relevant source code files with a different license, but this one should be representative of the the product as a whole baring a few exceptions like the version info files.